api/stream-legal-entity/openapi.yaml

```diff
@@ -451,6 +451,10 @@ components:
           $ref: '#/components/schemas/DebitAccount'
         accountHolderCountry:
           $ref: '#/components/schemas/AccountHolderCountry'
+        subscriptions:
+          type: array
+          items:
+            $ref: '#/components/schemas/Subscription'
       allOf:
         - $ref: '#/components/schemas/BaseProduct'
```

Comment:
For positive pay, subscriptions only need to work for current accounts, but it would be nice for others if it also works. Arrangement manager supports subscription for all arrangements. Can you consider defining these on the BaseProduct?

Reply:
@jjjasper Sure, it seems to be feasible.